conduit/pipeline/common.go

```diff
@@ -74,9 +76,19 @@ func Retries[X pluginInput, Y pluginOutput](f func(x X) (Y, error), x X, p *pipe
 	return
 }
 
-// RetriesNoOutput applies the same logic as Retries, but for functions that return no output.
-func RetriesNoOutput[X pluginInput](f func(x X) error, a X, p *pipelineImpl, msg string) (time.Duration, error) {
-	_, d, err := Retries(func(x X) (empty, error) {
+// TODO: probly the following function and its unit test should be axed
+// retriesNoInput applies the same logic as Retries, but for functions that take no input.
```

Comment:
![Uploading image.png…]()

Reply:
@algochoi - I had originally added this because I wanted to retry the call to `GetGenesis()` (which takes no params) inside of the pipeline's `Init()`. But then I realized that retrying at that point is too late and you'd really have have the importer do that (and it can't access this package).

UPSHOT: I commit to deleting this function and its test before merging, unless there's a use case that crops up mighty soon.